docs/source/features/metrics.md

```diff
@@ -57,6 +57,22 @@ You can set environment variable values on the command line as seen below, or wi
 ENGINE_API_KEY=YOUR_API_KEY ENGINE_SCHEMA_TAG=development node start-server.js
 ```
 
+### Debugging
+
+You can set the [`debugPrintReports` option](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/agent.ts#L429-L433) in the `engine` section of the Apollo Server constructor to see debug output regarding trace transmission.  For example:
```

Comment:
Apollo Server -> `ApolloServer`

Reply:
"see debug output regarding trace transmission" -> "automatically log debugging information for all reporting requests sent to Graph Manager"

Reply:
Resolved via 8b0f8136.